sdk/schemaregistry/schema-registry/test/public/schemaRegistry.spec.ts

```diff
@@ -64,8 +64,11 @@ describe("SchemaRegistryClient", function () {
   let client: SchemaRegistryClient;
   let schema: SchemaDescription;
 
-  beforeEach(function (this: Context) {
-    ({ client, recorder } = createRecordedClient(this));
+  beforeEach(async function (this: Context) {
+    recorder = new Recorder(this.currentTest);
+    await recorder.start(startOptions);
+    client = createRecordedClient(recorder);
+    if (!env.SCHEMA_REGISTRY_GROUP) throw new Error("SCHEMA_REGISTRY_GROUP is not defined");
```

Comment:
could this check be factored into a function since it is needed twice?

You could also grab the value of `env.SCHEMA_REGISTRY_GROUP` in a variable and reference that variable instead.

Reply:
updated